rpi-source

```diff
@@ -238,13 +255,13 @@ def rpi_update_method(uri):
         fw_rev = f.read().strip()
     info("Firmware revision: %s" % fw_rev)
 
-    repo_short = urlparse.urlparse(uri).path
+    repo_short = urllib.parse.urlparse(uri).path
 
     repo_api = "https://api.github.com/repos%s" % repo_short
     repo_raw = "https://raw.githubusercontent.com%s" % repo_short
 
-    kernel.git_hash = download("%s/%s/git_hash" % (repo_raw, fw_rev)).strip()
-    kernel.symvers = "%s/%s/Module%s.symvers" % (repo_raw, fw_rev, processor_type_suffix())
+    kernel.git_hash = download("%s/%s/git_hash" % (repo_raw, fw_rev)).strip().decode('UTF-8')
```

Comment:
Should probably use 'utf-8' here rather than 'UTF-8', for consistency with the other `decode` functions in this script?